lib/engine/logger.rb

```diff
@@ -0,0 +1,67 @@
+# frozen_string_literal: true
```

Comment:
not sure we need this logger class

Reply:
Everything could be included in the Game class, but it getting too big and this way the code will be more organised. This class will still grow a bit, because some lines of the logs will probably have relationships with other lines.